### PR TITLE
Admin CLI add delete visibility from ES

### PR DESCRIPTION
**Pull request by vancexu**
image
image

### CODE REVIEW

File: tools/cli/admin.go [hunk 1]
```diff
@@ -520,6 +524,42 @@ func newAdminElasticSearchCommands() []cli.Command {
...
+					Name:  FlagMuttleyDestinationWithAlias,
+					Usage: "Optional muttely destination to ElasticSearch cluster",
+				},
+				cli.StringFlag{
+					Name:  FlagIndex,
+					Usage: "ElasticSearch target index",
+				},
+				cli.StringFlag{
+					Name:  FlagInputFileWithAlias,
+					Usage: "Input file of indexer.Message in json format, separated by newline",
```
Comment (longquanzheng) [inline tools/cli/admin.go]: Please explain more detail about the file format

File: tools/cli/adminElasticSearchCommands.go [hunk 2]
```diff
@@ -202,6 +205,59 @@ func AdminIndex(c *cli.Context) {
...
+	ratelimiter := tokenbucket.New(rps, clock.NewRealTimeSource())
+
+	file, err := os.Open(inputFileName)
+	if err != nil {
+		ErrorAndExit("Cannot open input file", nil)
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+
+	scanner.Scan() // skip first line
```
Comment (longquanzheng) [inline tools/cli/adminElasticSearchCommands.go]: what is the first line and why it has to skip?

File: tools/cli/admin.go (hunk 1 above)
Comment (vancexu) [reply tools/cli/admin.go]: addressed

File: tools/cli/adminElasticSearchCommands.go (hunk 2 above)
Comment (vancexu) [reply tools/cli/adminElasticSearchCommands.go]: added in helper message. First line is list result table header, see top comments for use example

**Pull request merged**